includes/header.php

```diff
@@ -152,31 +152,10 @@ function gtag() {
             <a class="nav-link" href="/tools">Tools</a>
           </li>
           <li class="nav-item p-1 dropdown">
-            <a class="nav-link dropdown-toggle" href="/usage/introduction" role="button" data-bs-toggle="dropdown">Usage</a>
+            <a class="nav-link dropdown-toggle" role="button" data-bs-toggle="dropdown">Docs</a>
             <div class="dropdown-menu">
-              <a class="dropdown-item" href="/usage/introduction">Getting started</a>
-              <a class="dropdown-item" href="/usage/installation">Installation</a>
-              <a class="dropdown-item" href="/usage/configuration">Pipeline configuration</a>
-              <a class="dropdown-item" href="/usage/offline">Running offline</a>
-              <a class="dropdown-item" href="/usage/usage_tutorials">Usage tutorials</a>
-              <a class="dropdown-item" href="/usage/reference_genomes">Reference genomes</a>
-              <a class="dropdown-item" href="/usage/data_management">Data Management</a>
-              <a class="dropdown-item" href="/usage/troubleshooting">Troubleshooting</a>
-              <a class="dropdown-item" href="/usage/nextflow">Nextflow resources</a>
-            </div>
-          </li>
-          <li class="nav-item p-1 dropdown">
-            <a class="nav-link dropdown-toggle" href="/developers/guidelines" role="button" data-bs-toggle="dropdown">Developers</a>
-            <div class="dropdown-menu">
-              <a class="dropdown-item" href="/developers/guidelines">Guidelines</a>
-              <a class="dropdown-item" href="/developers/adding_pipelines">Adding a new pipeline</a>
-              <a class="dropdown-item" href="/developers/modules">Modules</a>
-              <a class="dropdown-item" href="/developers/release_checklist">Release checklist</a>
-              <a class="dropdown-item" href="/tools/docs/latest/pipeline_lint_tests/">Lint error codes</a>
-              <a class="dropdown-item" href="/developers/sync">Template synchronisation</a>
-              <a class="dropdown-item" href="/developers/developer_tutorials">Developer tutorials</a>
-              <a class="dropdown-item" href="/developers/editor_plugins">Code editor plugins</a>
-              <a class="dropdown-item" href="/developers/design_guidelines">Graphic design guidelines</a>
+              <a class="dropdown-item" href="/docs/usage/introduction">Usage</a>
+              <a class="dropdown-item" href="/docs/contributing/guidelines">Contributing</a>
```

Comment:
I'm not sure that this really adds much now.. It doesn't save a click most of the time. I think it's probably better to just have a flat link to the docs?

Reply:
Ideally a new page with some kind of overview, but for now could be `/docs/usage/introduction` I guess